src/bun.js/Weak.zig

```diff
@@ -4,6 +4,8 @@ const JSC = bun.JSC;
 pub const WeakRefType = enum(u32) {
     None = 0,
     FetchResponse = 1,
+    PostgreSQLQueryClient = 2,
+    HTTPRequest = 3,
```

Comment:
I don't think we can use weak

it's very slow

Reply:
I see will replace with strong

Reply:
Replaced with native WeakRef so JS side does not need to be alive longer